### ISSUE TITLE
go client: No Domain when polliing for task

### ISSUE THREAD

**Issue by rmohta** [Netflix/conductor#1952]
When using go client with an empty string domain, it's creating below GET call, which I think Conductor server does not like. Removing *domain=&* from the call works for polling. Is this is a known bug?

```
2020/11/02 14:19:23 Sending [ GET ] request to Server ( http://localhost:8080/api/tasks/poll/get_model?workerid=C02&domain=& ):
2020/11/02 14:19:23 Body:
2020/11/02 14:19:23 
2020/11/02 14:19:24 Received response from Server ( http://localhost:8080/api ):
2020/11/02 14:19:24 Status:  204 No Content
2020/11/02 14:19:24 Response:
2020/11/02 14:19:24 
```

Go client
github.com/netflix/conductor/client/go v0.0.0-20201029170655-6bce3912d3ce

**Comment by apanicker-nflx** [comment]
@djcass44 Could you please help with this potential issue in the go client? Thank you.

**Comment by djcass44** [comment]
@apanicker-nflx I've implemented a possible fix in #1976, are you able to take a look?
